src/main/java/seedu/guest/model/guest/Name.java

```diff
@@ -10,14 +10,16 @@
 public class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, apostrophes ('), hyphens (-), periods (.), "
-                    + "and commas.\nThey must start with an alphanumeric character, and cannot be blank.";
+            "Names should only contain alphabets, spaces, apostrophes ('), hyphens (-), periods (.), commas (,), "
+                    + "and forward slashes (/).\nThey must start with an alphabet, and cannot be blank.\n "
+                    + "They should not contain any field prefixes after whitespace in its content (e.g., "
+                    + "\"n/James p/Barnes\").";
 
     /*
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "^[\\p{Alnum}]([\\p{Alnum}',. -])*$";
+    public static final String VALIDATION_REGEX = "^[A-Za-z]([A-Za-z',. /-])*$";
```

Comment:
Just checking if names should end with alphabets as well, if not that is fine!

Reply:
It is possible to implement, but we should give more flexibility (e.g., `James D.`)

Reply:
Good note, agreed!